src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -215,6 +215,7 @@
   <Target Name="ArchiveTestBuild" AfterTargets="CreateAssemblyListTxt" Condition="'$(ArchiveTests)' == 'true'">
     <PropertyGroup>
       <TestArchiveDir>$(TestWorkingDir)$(OSPlatformConfig)/archive/tests/</TestArchiveDir>
+      <ProjectJson Condition="!Exists('$(ProjectJson)')">$(OriginalProjectJson)</ProjectJson>
```

Comment:
This is kind of hacky, but is currently necessary because we don't guarantee that we are actually going to generate a new project.json when we update the ProjectJson property.  It would be better if we had a property which we depended on that signaled if a new project.json was created or not.  I've created an issue to track fixing this.